htdocs/user/class/api_user.class.php

```diff
@@ -38,68 +38,66 @@ class AccountApi extends DolibarrApi
 	);
 
 	/**
-	 * @var User $account {@type User}
+	 * @var User $user {@type User}
 	 */
-	public $account;
+	public $useraccount;
 
 	/**
 	 * Constructor
 	 *
-	 * @url	account/
+	 * @url	user/
 	 * 
 	 */
 	function __construct() {
 		global $db, $conf;
 		$this->db = $db;
-		$this->account = new User($this->db);
+		$this->useraccount = new User($this->db);
 	}
 
 	/**
-	 * Get properties of an account object
+	 * Get properties of an user object
 	 *
-	 * Return an array with account informations
+	 * Return an array with user informations
 	 *
-	 * @param 	int 	$id ID of account
+	 * @param 	int 	$id ID of user
 	 * @return 	array|mixed data without useless information
 	 * 
-	 * @url	GET account/{user}
+	 * @url	GET user/{id}
 	 * @throws 	RestException
 	 */
 	function get($id) {
-		//if (!DolibarrApiAccess::$user->rights->user->lire)
-		//{
+		//if (!DolibarrApiAccess::$user->rights->user->user->lire) {
 			//throw new RestException(401);
```

Comment:
@eldy I'm talking about these authorization checks... For user table cannot find the right ones...